course_access_groups/acl_backends.py

```diff
@@ -1,19 +1,72 @@
 """
 Access Control backends to implement the Course Access Groups.
 """
-import six
 
+from openedx.core.djangoapps.site_configuration import helpers as configuration_helpers
+from organizations.api import get_course_organizations
+from organizations.models import UserOrganizationMapping
+from course_access_groups.models import (
+    CourseAccessGroup,
+    GroupCourse,
+    Membership,
+)
 
-def dummy_backend(user, resource, default_has_access, options):  # pylint: disable=unused-argument
+
+def is_organization_staff(user, course):
+    """
+    Helper to check if the user is organization staff.
+
+    TODO: Handle single-site setups in which organization is not important
+    """
+    # TODO: What if a course has two orgs? data leak I guess?
+    organizations = get_course_organizations(course.id)
```

Comment:
@OmarIthawi A course can belong to multiple organizations?

Reply:
I hate mysterious exceptions (such as `objects.get()` complaining that two objects exists) that has nothing to do with my code

Therefore I tend to code safely and allow for some discrepancy just to avoid 500 errors.